webapp/src/blocks.tsx

```diff
@@ -2499,19 +2508,16 @@ function isBreakpointSet(block: Blockly.BlockSvg) {
 }
 
 function shouldEventHideFlyout(ev: Blockly.Events.Abstract) {
-    if (ev.type === "var_create" || ev.type === "marker_move" || ev.type === "toolbox_item_select") {
+    if (
+        ev.type === Blockly.Events.VAR_CREATE ||
+        ev.type === Blockly.Events.TOOLBOX_ITEM_SELECT ||
+        ev.type === Blockly.Events.BLOCK_DRAG ||
+        // Selected events are fired late when using 'T' to open the toolbox during a keyboard-driven block move.
+        ev.type === Blockly.Events.SELECTED
+    ) {
```

Comment:
Not certain about this change. See comment in code for the reason behind the change. I think that now the flyout closes when it loses focus, this is probably fine.